src/main/java/io/rpg/model/object/GameObject.java

```diff
@@ -46,6 +46,8 @@ public class GameObject implements GameObjectStateChange.Emitter {
   @NotNull
   private final Set<GameObjectStateChange.Observer> stateChangeObservers;
 
+  private int strength;
+
```

Comment:
I think we should start moving more towards generic properties. This has to be added to Jira (I believe there isn't any task for this right now).